### ISSUE TITLE
occu_mask

### ISSUE THREAD

**Issue by lt573152873** [lliuz/ARFlow#41]
during training, Occu_mask whether there is gradient return? when occu_mask is zeros, the loss is small; 

if need    occu_mask -> occu_mask.detach()  ?
